agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -17,7 +17,7 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 group=default
 dir="/tmp"
 sysinfo_opts_default=( block libvirt kernel_config sos topology )
-sysinfo_opts=${sysinfo_opts_default[*]}
+sysinfo_opts="${sysinfo_opts_default[*]} ara"
```

Comment:
@portante I think ara shouldn't be a default option, it should be collected only when all is passed as an argument. That's the reason why we have sysinfo_opts_default and sysinfo_opts right?